prinz/src/main/scala/pl/touk/nussknacker/prinz/util/collection/immutable/VectorMultimap.scala

```diff
@@ -2,7 +2,7 @@ package pl.touk.nussknacker.prinz.util.collection.immutable
 
 import scala.collection.mutable
 
-class VectorMultimap[K, V](private val delegate: mutable.LinkedHashMap[K, Vector[V]]) {
+class VectorMultimap[K, V](private val delegate: mutable.Map[K, Vector[V]]) {
```

Comment:
I decided to use LinkedHashMap because it keeps key ordering Wont it be a problem if we lose column ordering in more complex examples?

Reply:
LinkedHashMap is back :)